### DISCUSSION TITLE
Does "enable DNS over HTTPS" option of Firefox can be set while using DNSCrypt in default setting ?

### DISCUSSION THREAD

**Discussion by Nokia808** [DNSCrypt/dnscrypt-proxy#1792] [Q&A]
Hello !

1st of all, I would like to clarify that I'm completely new to "DNS over HTTPS" ........ Till now I'm only depending on DNSCrypt with it's default setting since some years ago .......

I'm installing DNSCrypt from official Fedora repositories, & already activated it, & it is currently working without problem on my Fedora 34 X64 bit Cinnamon edition without problem. I'm already set my /etc/resolv.conf is already edited by me to contain the following:

nameserver 127.0.0.1
options edns0

My questions is as following: my Firefox has "enable DNS over HTTPS" option - go to "Preference" | "General settings" | "Network settings" | "Settings" | ...........

1) does activating this Firefox option will working while DNSCrypt is already activated on my system or not ? If not what I have to do to utilize DoH ?

2) currently my Firefox set at "Use system proxy settings". Is "Enable DNS over HTTPS" option is compatible with "Use system proxy setting" or I need to change "Use system proxy setting" to other choice ?

Thank you.

**Comment by jedisct1** [comment]
If your system has already been configured to use `dnscrypt-proxy` as a resolver, there's no need to fiddle with Firefox's settings.

If you really want to use this, see [local DoH](https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Local-DoH). Unless you want to participate in the experiment, this is useless.

**Comment by Nokia808** [comment]
@jedisct1 
Dear, thank you for response & orientation !
Frankly speaking, I asked this issue because today I discovered & installed a long time waited Firefox add-on for DNSSEC validation which it is:
https://addons.mozilla.org/en-US/firefox/addon/dnssec-dane-validator/

As you see from description of this add-on, it says that it use DoH for it jobs ...... For that I'm asked ...... I did not change any of my setting (DNSCrypt is currently active + all my system traffic run through DNSCrypt + & my Firefox using "system default proxy setting"). The add-on seem to be working fine & from your response above "no need to fiddle with Firefox's settings" it seem that this add-on has no need to use "enable DoH" setting of Firefox ......
